packages/vue/src/components/confirm-reset-password.vue

```diff
@@ -57,15 +51,15 @@ const onLostYourCodeClicked = (): void => {
 };
 
 const onInput = (e: Event) => {
-  const { name, value } = <HTMLInputElement>e.target;
+  const { name, value } = e.target as HTMLInputElement;
```

Comment:
The linter did not like `<HTMLInputElement>` so I swapped it out to `as HTMLInputElement`